### PR TITLE
feat(edit): global preview

### PR DESCRIPTION
**Pull request by natashatikhonova**
### Description:
- removed preview for each tile
- show preview for the entire board (exapandable)
- update styling of edit page
### Images:
image
image
image

### CODE REVIEW

**Review by oyvindgrutle — APPROVED** [review]
LGTM

File: next-tavla/src/Board/scenarios/QuayTile/index.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export function QuayTile({
     whitelistedLines,
     whitelistedTransportModes,
     columns,
-}: TQuayTile) {
+}: TQuayTile & { preview?: boolean }) {
```
Comment (lindtvedtsebastian) [inline next-tavla/src/Board/scenarios/QuayTile/index.tsx:15]: Denne brukes vel ikke til noe?

File: next-tavla/src/Board/scenarios/StopPlaceTile/index.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export function StopPlaceTile({
     whitelistedLines,
     whitelistedTransportModes,
     columns,
-}: TStopPlaceTile) {
+}: TStopPlaceTile & { preview?: boolean }) {
```
Comment (lindtvedtsebastian) [inline next-tavla/src/Board/scenarios/StopPlaceTile/index.tsx:14]: Denne brukes vel heller ikke til noe?

File: next-tavla/src/Board/scenarios/Board/index.tsx [hunk 3]
```diff
@@ -6,16 +6,22 @@ import classes from './styles.module.css'
...
-            return <StopPlaceTile {...tileSpec} />
+            return <StopPlaceTile {...tileSpec} preview={preview} />
         case 'quay':
-            return <QuayTile {...tileSpec} />
+            return <QuayTile {...tileSpec} preview={preview} />
     }
 }
 
-function Board({ board }: { board: TBoard }) {
+function Board({ board, preview }: { board: TBoard; preview?: boolean }) {
```
Comment (lindtvedtsebastian) [inline next-tavla/src/Board/scenarios/Board/index.tsx:24]: tror ikke vi trenger `preview` proppen

File: next-tavla/src/Board/scenarios/Board/index.tsx [hunk 4]
```diff
@@ -6,16 +6,22 @@ import classes from './styles.module.css'
...
+    tileSpec: TTile
+    preview?: boolean
+}) {
     switch (tileSpec.type) {
         case 'stop_place':
-            return <StopPlaceTile {...tileSpec} />
+            return <StopPlaceTile {...tileSpec} preview={preview} />
         case 'quay':
-            return <QuayTile {...tileSpec} />
+            return <QuayTile {...tileSpec} preview={preview} />
```
Comment (lindtvedtsebastian) [inline next-tavla/src/Board/scenarios/Board/index.tsx:20]: preview prop

**Review by lindtvedtsebastian — CHANGES_REQUESTED** [review]

**Review by lindtvedtsebastian — APPROVED** [review]

**Pull request merged**